src/data/api/client.js

```diff
@@ -3,16 +3,7 @@ import Cookies from 'js-cookie';
 import 'whatwg-fetch'; // fetch polyfill
 
 import endpoints from './endpoints';
-
-/*
-API returns snake-cased attribute names, but API expects names that match database attribute names,
-which are inconsistently cased
-*/
-const assetAPIAttributesToDatabaseAttributes = {
-  display_name: 'displayname',
-  content_type: 'contentType',
-  date_added: 'uploadDate',
-};
+import { getDatabaseAttributesFromAssetAttributes } from '../../utils/getAttributes';
```

Comment:
I'm pulling this functionality out into a utils file, since I found a need for something very similar elsewhere.